tools/tpm2_akparse.c

```diff
@@ -94,7 +94,9 @@ static bool save_alg_and_key_to_file(const char *key_file, UINT16 alg_type,
             goto out;
         }
 
+        tpm2_tool_output("- ");
         tpm2_util_print_tpm2b(tmp);
+        tpm2_tool_output("\n");
```

Comment:
This output is flaky. Have it map to something like:
ECC:
```
algorithm: ecc
  x : fdjhdfkhj
  y: djhjfhg
```
RSA
```
algorithm: rsa
   foo: bar
```
Not sure what the rsa sub-name is offhand. Writing this so I don't forget TODO tomorrow.